libs/blocks/graybox-promote/components/graybox-promote-lit.js

```diff
@@ -128,19 +127,19 @@ class GrayboxPromote extends LitElement {
     super();
 
     this.spLogin = async () => {
-      const scopes = ['files.readwrite', 'sites.readwrite.all'];
-      const extraScopes = [`${origin}/.default`];
-          //TODO - delete below
-          this.spToken = '1234';
-          return null;
-          //TODO - delete above
-      return login({ scopes, extraScopes, telemetry: TELEMETRY })
-        .then(() => {
-          this.spToken = accessToken.value || accessTokenExtra.value;
-        })
-        .catch((error) => {
-          throw error;
-        });
+      // TODO - delete below
+      this.spToken = '1234';
+      return null;
+      // TODO - uncomment below
+      // const scopes = ['files.readwrite', 'sites.readwrite.all'];
+      // const extraScopes = [`${origin}/.default`];
+      // return login({ scopes, extraScopes, telemetry: TELEMETRY })
+      //   .then(() => {
+      //     this.spToken = accessToken.value || accessTokenExtra.value;
+      //   })
+      //   .catch((error) => {
+      //     throw error;
+      //   });
```

Comment:
Still needed?

Reply:
I'll need it for my next ticket. I commented it out to avoid the eslint check failing.